src/copyToClipboard.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+export default function copyToClipboard(text: string) {
+    const el = document.createElement('textarea');
+    el.value = text;
+    el.style.position = 'absolute';
+    el.style.left = '-9999px';
+
+    document.body.appendChild(el);
+    el.select();
+
+    let success;
```

Comment:
You can avoid creating a `success` variable by returning the success value directly and removing the child in a `finally` block:
```ts
try {
    return document.execCommand('copy');
} catch (err) {
    console.error(err);
    return false;
} finally {
    document.body.removeChild(el);
}
```

Reply:
Yeah this was like 99% copy pasta from a Sindre module that had extra cruft I didn't want for mobile devices 😝